libs/aws/tests/integration_tests/chat_models/test_standard.py

```diff
@@ -44,6 +39,7 @@ def standard_chat_model_params(self) -> dict:
         return {
             "temperature": 0,
             "max_tokens": 100,
+            "stop_sequences": [],
```

Comment:
We shouldn't include `stop_sequences` here, as it's not a standard ChatModel parameter: https://python.langchain.com/docs/concepts/chat_models/#standard-parameters